curriculum/js1/8.js

```diff
@@ -5,8 +5,20 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, min = '') => {
```

Comment:
you can directly set default values in the params. `solution = (a,b,min=Math.min(a,b))`

Reply:
this way, you can remove the next line